### PR TITLE
Admin management implementation of task1&2

### PR DESCRIPTION
**Pull request by upendra2610**
i have implemented the api and interface for service .
Let me know if i am doing in correct way.

### CODE REVIEW

File: admin-management-service/src/main/java/com/scaler/adminmanagementservice/controllers/AdminController.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import com.scaler.adminmanagementservice.dtos.GenericAdminDto;
+import com.scaler.adminmanagementservice.services.AdminService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/admins")
+public class AdminController {
```
Comment (ajeetcloud) [inline admin-management-service/src/main/java/com/scaler/adminmanagementservice/controllers/AdminController.java:12]: Please create interface - https://www.baeldung.com/spring-interface-driven-controllers#2-interface

File: admin-management-service/src/main/java/com/scaler/adminmanagementservice/models/AdminModel.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.scaler.adminmanagementservice.models;
+
+public class AdminModel {
+    private Long id;
```
Comment (ajeetcloud) [inline admin-management-service/src/main/java/com/scaler/adminmanagementservice/models/AdminModel.java:4]: how are you going to access the private fields

File: .idea/sonarlint/issuestore/8/e/8ec9a00bfd09b3190ac6b22251dbb1aa95a0579d
Comment (ajeetcloud) [inline .idea/sonarlint/issuestore/8/e/8ec9a00bfd09b3190ac6b22251dbb1aa95a0579d:1]: remove unrequired files from the PR.

File: admin-management-service/src/main/java/com/scaler/adminmanagementservice/models/AdminModel.java (hunk 2 above)
Comment (upendra2610) [reply admin-management-service/src/main/java/com/scaler/adminmanagementservice/models/AdminModel.java:4]: we can access via getter setter when we use it , will annotate it by @Getter and @Setter

**Pull request closed without merging**